_includes/usagepolicy.html

```diff
@@ -0,0 +1,32 @@
+<div class="full-width-bg component" xmlns="http://www.w3.org/1999/html">
+  <div class="grid-wrapper">
+    <div class="width-6-12 width-12-12-m">
+      <p>This page contains the policy for the Red Hat sponsoered anonymous data collection. See <a href="/usage">Usage</a> page for details.</a></p>
+      <p class="intropara">In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat and community extension writers to understand how the software is being used and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may be shared internally within Red Hat to improve the user experience.</p>
+      <H3>What information does Red Hat collect?</H3>
+      <p>Tools within the software monitor various metrics and this information is transmitted to Red Hat.  Metrics include the following information such as:</p>
```

Comment:
This sentence is weird: `Metrics include the following information such as`. `following` + `such as` doesn't sound good to me.
Also I think we need to be comprehensive as to the nature of the data collected, so I wouldn't use `such as`.

Reply:
this is how others are defined by legal. we can and should definitely post examples but I think its explicit put here to not require updating it on every small change or adjustment we ne need.